pennylane/devices/execution_config.py

```diff
@@ -43,7 +43,8 @@ class MCMConfig:
     def __post_init__(self):
         """Validate the configured mid-circuit measurement options."""
         if self.postselect_mode not in ("hw-like", "fill-shots", "pad-invalid-samples", None):
-            raise ValueError(f"Invalid postselection mode '{self.postselect_mode}'.")
+            raise ValueError(
+                f"Invalid postselection mode '{self.postselect_mode}'.")
```

Comment:
No need to update the formatting here :)
```suggestion
            raise ValueError(f"Invalid postselection mode '{self.postselect_mode}'.")
```